tools/mason/MasonUpdate.chpl

```diff
@@ -196,7 +196,23 @@ proc parseChplVersion(brick: borrowed Toml?): (VersionInfo, VersionInfo) {
   var low, hi : VersionInfo;
 
   try {
-    const formatMessage = "\n\n" +
+    var res = checkChplVersion(chplVersion, low, hi);
+    low = res[0];
+    hi = res[1];
+  } catch e : Error {
+    const name = brick!["name"]!.s + "-" + brick!["version"]!.s;
+    stderr.writeln("Invalid chplVersion in package '", name, "': ", chplVersion);
+    stderr.writeln("Details: ", e.message());
+    exit(1);
+  }
+
+  return (low, hi);
+}
+
+proc checkChplVersion(chplVersion, low, hi) throws {
```

Comment:
How about we use `low, high` and `lo, hi` for arg/variable names, respectively? This lets us stick to typical Chapel style of camelCase vars